survey/tests.py

```diff
@@ -1,6 +1,36 @@
-from django.test import TestCase
+from django.test import TestCase, Client
+from django.conf import settings
+from django.utils.importlib import import_module
 import models
+import datetime
```

Comment:
That worked 👍 . Since there seem to be no other issues, I'll rebase and merge this in next.